src/components/Button.vue

```diff
@@ -0,0 +1,35 @@
+
+<template>
+  <button class="btn" @click="$emit('onClick')">
+    {{ message }}
+  </button>
+</template>
+
+<script>
+export default {
+  name: "Button",
+  props: {
+    message: { required: true, type: String },
+  },
+  emits: ["onClick"],
+};
+</script>
+
+<style scoped>
+.btn {
+  height: 50px;
+  width: 220px;
+  background-color: #4caf50;
```

Comment:
We should consider how the color would work with theming

Reply:
Tried adding Vuetify for themes, but its a beta addition for Vue3 and there were several issues with it. Will add this once official release will be done. Other options are not standard.